### PR TITLE
[COM-721] ColumnLayout block

### PR DESCRIPTION
**Pull request by guilean**
**Checklist**

- [x] New feature (non-breaking change which adds functionality)

**Please check if the PR fulfills these requirements**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Storybook or docs have been added / updated (for bug fixes / features)

**What is the ticket / issue associated with this change?**
https://impulsumvc.atlassian.net/browse/COM-721

### CODE REVIEW

File: packages/blocks/src/MediaBlock/types.ts [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
   variant?: MediaBlockVariant;
+  bounded?: boolean;
   alignItems?: 'start' | 'center' | 'end';
 }
 
 export interface MediaBlockStaticMembers {
   Image: FC<ImageProps>;
   TextPairing: TextPairingType;
+  Tag: typeof Tag;
+  Link: FC<LinkProps>;
```
Comment (tkanzakic) [inline packages/blocks/src/MediaBlock/types.ts:17]: typeof Link?

File: packages/blocks/src/index.ts [hunk 2]
```diff
@@ -1,8 +1,10 @@
 export * from './AccordionGallery';
 export * from './BrandLogos';
+export * from './ColumnLayout';
 export * from './CookieBanner';
 export * from './Footer';
 export * from './Hero';
+export * from './MediaBlock';
```
Comment (tkanzakic) [inline packages/blocks/src/index.ts:7]: 👌

File: packages/blocks/src/ColumnLayout/ColumnLayout.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Flex, Link, TextPairing } from '@cmpsr/components';
+import { ColumnLayoutType } from './types';
+import { MediaBlock } from '../';
+
+export const ColumnLayout: ColumnLayoutType = (props) => <Flex flexDirection="column" {...props} />;
+
+ColumnLayout.ColumnGroup = Flex;
+ColumnLayout.Column = MediaBlock;
```
Comment (tkanzakic) [inline packages/blocks/src/ColumnLayout/ColumnLayout.tsx:9]: we should the variant to vertical by default, in here or in the `MediaBlock` component

File: packages/blocks/src/ColumnLayout/ColumnLayout.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+        </ColumnLayout.Column>
+      )}
+    </ColumnLayout.ColumnGroup>
+  </ColumnLayout>
+);
+
+export const Playground = Template.bind({});
+Playground.args = {
+  label: 'The next generation of care for women and families',
+  subLabel: 'We’re setting a new standard of care for families across geographies, cultures, and backgrounds.',
```
Comment (tkanzakic) [inline packages/blocks/src/ColumnLayout/ColumnLayout.stories.tsx:166]: we should use a different copy (applies to the whole PR)

Comment (guilean) [reply packages/blocks/src/ColumnLayout/ColumnLayout.stories.tsx:166]: Modified all copies of `care for women and families...`. We should update the stories once we have final designs for all blocks

File: packages/blocks/src/ColumnLayout/ColumnLayout.tsx (hunk 3 above)
Comment (guilean) [reply packages/blocks/src/ColumnLayout/ColumnLayout.tsx:9]: Updated `MediaBlock` to have `vertical` as default value of `variant`

File: packages/blocks/src/MediaBlock/types.ts (hunk 1 above)
Comment (guilean) [reply packages/blocks/src/MediaBlock/types.ts:17]: 👍🏻

File: packages/blocks/package.json [hunk 5]
```diff
@@ -26,7 +26,7 @@
     "type:emit": "tsc --project ./tsconfig.json -d --declarationDir lib --declarationMap --emitDeclarationOnly && find lib -type f \\( -name '*.test.*' -o -name '*.stories.*' \\) -exec rm {} +"
   },
   "peerDependencies": {
-    "@cmpsr/components": "^4.0.0",
+    "@cmpsr/components": "^4.3.0",
```
[the remark this answers is not in the document]
Comment (guilean) [reply packages/blocks/package.json:29]: I believe this bump was done manually to have the latest from components package

File: packages/blocks/src/ColumnLayout/types.ts [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import { FC } from 'react';
+import { FlexProps, TextPairingType, Link, Flex } from '@cmpsr/components';
+import { MediaBlock } from '../';
+
+export interface ColumnLayoutStaticMembers {
+  TextPairing: TextPairingType;
```
[the remark this answers is not in the document]
Comment (guilean) [reply packages/blocks/src/ColumnLayout/types.ts:6]: Renamed

**Review by tkanzakic — APPROVED** [review]

**Pull request merged**